exe/floe

```diff
@@ -33,6 +34,6 @@ runner_options = opts[:docker_runner_options].to_h { |opt| opt.split("=", 2) }
 
 Floe::Workflow::Runner.docker_runner = runner_klass.new(runner_options)
 
-output = workflow.run!
+workflow.run!
 
-puts output.inspect
+puts workflow.context.state["Output"].inspect
```

Comment:
grumble: I wish it had returned the output.
But I guess the `workflow` has always been stateful

Reply:
Yeah I was surprised that `run` returned `self` but that is actually from Jason's commit https://github.com/ManageIQ/floe/pull/33/files#diff-7ab660aebba40fbc9105ec3f6fe80d20b0d0f5d368dc8a7c3db8294c1e91a077R64